src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -148,6 +150,12 @@ const CreateImageWizard = ({ startStepIndex = 1 }: CreateImageWizardProps) => {
     selectBlueprintDescription(state)
   );
   const activationKey = useAppSelector((state) => selectActivationKey(state));
+  const partitions = useAppSelector((state) => selectPartitions(state));
+  const [isNextDisable, setNextDisable] = useState(false);
```

Comment:
Very nit picky, but can we update the names of the variables to `isNextDisabled` and `setNextDisabled` ?

```suggestion
  const [isNextDisabled, setNextDisabled] = useState(false);
```